plugin/active_request.py

```diff
@@ -0,0 +1,73 @@
+from .core.sessions import SessionViewProtocol
```

Comment:
Shouldn't this file live in `core/` since it only deals with core types?

Reply:
@rwols you want me to address that?

Reply:
No let me do it :D I'll get to it this evening, I hope.